### DISCUSSION TITLE
Remove Spaces in Output PNG

### DISCUSSION THREAD

**Discussion by jestapinski** [FRSOURCE/cypress-plugin-visual-regression-diff#189] [Q&A]
Hello! Enjoying the library thus far!

I'm wondering if it is possible to remove all spaces in an output filename for a png. I noticed the `title` field, but modifying that doesn't seem to get all of the spaces eradicated. Is there some flag I'm missing perhaps to replace all spaces in an output file with dashes perhaps? (Including #0)

**Comment by FRSgit** [comment]
Hey! It is possible now (with version 4.0.0)! Have a look at the README for details :)
